css/index.css

```diff
@@ -1,3 +1,490 @@
+  cursor: pointer;
+  border-radius: 2px;
+  padding: 0.63em;
+  margin: 0 4px;
+  height: 2.19em;
+  width: 6.25em;
+  border-style: none;
+  font-weight: bolder;
+  font-size: 0.88em;
+  text-transform: uppercase;
+  display: inline-block;
+}
+
+.show-more-text {
+  color: #030303;
+  width: 100%;
+  height: auto;
+  font-weight: 400;
+  font-size: 0.9em;
+  line-height: 1.6em;
+  display: inline-block;
+}
+
+.show-more {
+  text-transform: uppercase;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 0.8em;
+  display: inline-block;
+  cursor: pointer;
+  color: #606060;
+  margin-bottom: 2em;
+}
+
+.aside-img {
+  display: inline-block;
+  width: 10em;
+  height: auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: row;
+  padding: 0.9em 0;
+  border-bottom: 1px solid #eee;
+}
+
+.views {
+  font-size: 13px;
+  color: #606060;
+}
+
+.text-container {
+  margin-left: 0.63em;
+  font-size: 0.83em;
+  line-height: 1.37em;
+}
+
+.svg {
+  width: 1.5em;
+  height: 1.53em;
+  background-color: rgb(255, 255, 255);
+  border-style: none;
+}
+
+.text {
+  color: #030303;
+  font-weight: 500;
+  font-size: 1.1em;
+}
+
+.title {
+  padding: 20px 0 8px;
+  border-bottom: 1px solid #eee;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: #606060;
+  width: 100%;
+  align-items: center;
+}
+
+.right-icon {
+  font-weight: 400;
+  font-size: 13px;
+  display: flex;
+  justify-content: space-between;
+  margin: 12px;
+  width: 60%;
+}
+
+.left-icon {
+  width: 30%;
+  font-size: 14px;
+}
+
+.icon-container {
+  width: 18%;
+  display: flex;
+  align-items: center;
+}
+
+/* Comment section */
+.comment-section {
+  border-top: 1px solid #eee;
+}
+
+.sort {
+  color: #606060;
+  font-size: 14px;
+  width: 50%;
+}
+
+.comment {
+  display: flex;
+  width: 50%;
+}
+
+.comments {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.comment-head {
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+  font-size: 0.98em;
+  justify-content: space-between;
+  margin: 1.18em 0;
+}
+
+.img-radius {
+  display: inline-block;
+  background-color: rgb(240, 7, 7);
+  color: rgb(255, 255, 255);
+  border-radius: 50%;
+  width: 4.9rem;
+  height: 4.9rem;
+  font-size: 2.9em;
+  text-align: center;
+}
+
+.add-comment {
+  font-size: 14px;
+  color: #606060;
+  font-weight: 400;
+  border-bottom: 1px solid #eee;
+  padding-bottom: 5px;
+  width: 90%;
+  align-self: center;
+}
+
+.other-comment {
+  width: 80%;
+  display: flex;
+  flex-direction: row;
+  font-size: 0.98em;
+  justify-content: space-between;
+  margin: 1.18em 0;
+}
+
+.comment-image {
+  width: 2.9em;
+  height: 2.9em;
+  border-radius: 50%;
+}
+
+.comment-img {
+  width: 3em;
+  border-radius: 50%;
+}
+
+.comment-block {
+  color: #030303;
+  font-size: 13px;
+  margin-left: 9px;
+}
+
+.comment-reply,
+.comment-replies {
+  margin: 0.987em 0;
+}
+
+/* Media queries */
+
+@media screen and (max-width: 994px) {
```

Comment:
Let's arrange the media query according to their sizes.

Reply:
Done.